src/thx/DateConst.hx

```diff
@@ -0,0 +1,18 @@
+package thx;
+
+class DateConst {
+  public inline static var millisPerSecond : Float = 1000;
+  public inline static var millisPerMinute : Float = millisPerSecond * 60;
```

Comment:
To be used for any high-resolution date handling, this will need to account for leap seconds at some point in the future. The number of millis per minute is not a constant.